sonar-helm-for-iac/src/main_test.go

```diff
@@ -76,29 +51,14 @@ func Test_exit_code_with_serialization_error(t *testing.T) {
 	stdin, _ := cmd.StdinPipe()
 	defer stdin.Close()
 	cmd.Start()
-	stdin.Write([]byte("foo.yaml\n1\napiVersion: v1\nvalues.yaml\n0\n\nEND\n"))
+	stdin.Write([]byte("\x00\x00\x00\x08foo.yaml\x00\x00\x00\x0EapiVersion: v1\x00\x00\x00\x01\x00\x00\x00\x0Bvalues.yaml\x00\x00\x00\x00"))
 	err := cmd.Wait()
```

Comment:
Is the trailing 0 something for this test only or not? I didn't see it neither on the Java side nor in input handling. Or did I miss something?

Reply:
The trailing 0 (4 bytes) is the content size of values.yaml